giraffe/src/utils/fromFlux.test.ts

```diff
@@ -1053,8 +1083,6 @@ describe('fastFromFlux', () => {
     expect(() => {
       fastFromFlux(CSV)
     }).not.toThrow()
-    const actual = fastFromFlux(CSV)
-    expect(actual.error).toBeTruthy()
```

Comment:
Maybe we can use this somehow? 

https://www.papaparse.com/docs#errors

Reply:
Great point, this CSV shouldn't actually throw an error, but it did in the previous implementation. This particular test was added since the old parser couldn't handle CSV partials and it caused the app to throw an error as a result. With papaparse, we can totally do that now, and I'm hoping it'll allow us to stream CSV partials in the future so that we don't have to worry about those types of issues going forward